pdu.c

```diff
@@ -455,6 +458,14 @@ static int pdu_parse_number(char ** pdu, size_t * pdu_length, unsigned digits, i
 			char digit;
 			if(*toa == NUMBER_TYPE_INTERNATIONAL)
 				*number++ = '+';
+			// BEGIN oioki proposed patch 2013-07-24
+			if(*toa == NUMBER_TYPE_ALPHANUMERIC)
```

Comment:
Please drop the `// BEGIN` and `// END` comments. They don't make sense after merging.

And make the `if` above into an `else if`. (Add braces around the first `if` too.)